pytest/tests/mocknet/run_adversenet.py

```diff
@@ -83,13 +83,39 @@ def check_slow_blocks(initial_metrics, final_metrics):
 
 def override_config(node, config):
     # Add config here depending on the specific node build.
-    pass
-    """
-    if "bad" in node.instance_name:
-        config["adversarial"] = {
-            "produce_duplicate_blocks": True
+
+    # Enable the following if you want to increase the block time
+    # from the default 0.6s to 1s.
+    if False:
+        config["consensus"]["min_block_production_delay"] = {
+            "secs": 1,
+            "nanos": 0,
+        }
+
+    # Enable the following if testing single-shard tracking.
+    if False:
+        config["state_sync"] = {
+            "dump": {
+                "location": {
+                    "GCS": {
+                        "bucket": "near-mocknet-snet2-state"
+                    }
+                },
+                "iteration_delay": {
+                    "secs": 1,
+                    "nanos": 0
+                },
+            }
+        }
+        config["store"]["state_snapshot_enabled"] = True
```

Comment:
No need to enable state snapshot for non dumping nodes. 
When you enable state snapshot for the dumping node, I suggest to avoid enabling compaction. The node will take a hit on the storage but at least it will not OOM every epoch.